src/lazy_map/core.clj

```diff
@@ -409,7 +416,7 @@
 
 (defn lazy-map
   "Constructs a lazy map from a map that may contain delays for values."
-  ([] (->LazyMap {}))
+  ([] (->LazyMap {} nil))
```

Comment:
Hmm, `->LazyMap` is a function generated by `deftype`. It used to be 1-arity because there was only one field on the type. There are now two fields on the type and so calling -`>LazyMap` with one argument is invalid. It's the only breaking change, but consumers should never have been calling `->LazyMap` directly anyways, instead preferring to use the lazy-map function provided here.

Reply:
I understand why we needed to update our wrapper around `->LazyMap`. I was wondering why `lazy-map` has a 0-arity implementation that always throws an exception. I think it's vim + fireplace as I can call 0-arity `lazy-map` in the repl but fireplace is throwing a NullPointerException. Same thing happens if I call `(lazy-map {})`
It's strange. I'll look into it. Still not a blocker.